step-api/src/main/java/org/jenkinsci/plugins/workflow/steps/AbstractSynchronousNonBlockingStepExecution.java

```diff
@@ -61,21 +78,4 @@ private static synchronized ExecutorService getExecutorService() {
         return executorService;
     }
 
-    private static class StepRunner implements Runnable {
```

Comment:
Seemed easier and clearer to me to use a `final` local variable used close to its declaration, and let `javac` take care of the rest.